### DISCUSSION TITLE
Undefined command in region_surface_areas example

### DISCUSSION THREAD

**Discussion by AdelaHlobilova** [PMEAL/porespy#530] [Q&A] [answered]
Hello all,

while running the example on "region_surface_areas" function listed on page https://porespy.org/examples/metrics/howtos/region_surface_areas.html, I encountered an undefined command, expressed as `regions = r.regions`, listed in line 4 in [2]. Could you please specify where does `r.regions` originate from as is it is not clear from the commands listed above it?

Best,
Adela

**Comment by jgostick** [comment, marked as answer]
We are currently in the process of transitioning V2.0.  The docs are built from the latest dev version (2.0), but the published packages on pip and conda are still 1.3.1. If you have installed from one of these places, then you'll be a bit out of date/incompatible.  This transition will be over soon (within a week or so).  The specific issue you mention above can probably be remedied by setting ``return_all=True`` in the ``snow_partitioning`` call.  In future versions we just always return all as a tuple so this argument has been removed.
